cli/azd/pkg/tools/internal/tools_util_functions.go

```diff
@@ -0,0 +1,45 @@
+package internal
```

Comment:
Was there some reason to move these into an `internal` pacakge vs keeping them in the `tools` package and just exporting them?  Did that help with some dependency management?

Reply:
I though we generally prefer using `internal` pattern instead of exporting all functions.

I don't have a preference. I just saw there was an `internal` folder already there and didn't want to export what it was not exported before.

I will move to exporting..  I'm fine with both approaches.  

Actually, the `internal` pattern is more typically for libraries, than for Applications.
But on the other hand, we could start considering the `pkg` folder as a library/API/backend which could be re-used for another clients...  (just saying)